lib/src/working_copy.rs

```diff
@@ -454,8 +453,9 @@ impl TreeState {
             // ignore it.
             return Ok(());
         }
+        let metadata = disk_path.symlink_metadata().unwrap();
```

Comment:
Do we want to keep these unwraps?

Reply:
I think I preserved the behavior in this commit, but they unwraps are gone in a few commits later ("working_copy: improve error handling when getting file stats").